alexsg2/Practice_10k_Music_App
# Adding redux for profile

What:
When the user registers the profile data is added to the firestore database. Also the profile data is added to redux. Also when the user logins back is the app fetches their profile data and adds it to redux.

How to test:
Create user
Navigate to profile
The profile page should have the data entered
Logout
Log in
Navigate to profile
The profile page should have the data entered
